client/src/signUp.js

```diff
@@ -0,0 +1,53 @@
+import React from "react"
+
+export default class SignUp extends React.Component {
+  state = {
+    username: "",
+    email: "",
+    password: "",
+    passwordConfirm: ""
+  }
+  handleSubmit = e => {}
+
+  handleChange = e => {
+    this.setState({
+      [e.target.id]: e.target.value
```

Comment:
Any reason why you switched to using `id` in this component, when in `login` you used name? I think it'd be better to keep consitancy. Nitpicky, i know, but just a thought.

Reply:
Agreed with Alex!